include/subnet_cadmium.hpp

```diff
@@ -1,7 +1,17 @@
-/**
-* Cristina Ruiz Martin
-* ARSLab - Carleton University
-*
+/** \brief This header file implements the Subnet class
+ *
+ * The subnet receives the message on input port
+ * and transmits the same message to the output port
+ * after generating some delay on output port.
+ *
+ * The subnet has two phases: active and passive
+ * it is in active state when transmitting the
+ * data and goes to passive state when no data
+ * is being send or acknowledged
+ *
```

Comment:
You may consider dividing this section into two halves. One for detailed/brief description and the other for author details.

Reply:
done